src/ui/MD/md_test.ts

```diff
@@ -0,0 +1,58 @@
+// This file contains almost all features that markdown ever uses. It is split by section to be better able to test different features.
```

Comment:
This shouldn't be called `md_test.ts` IMO, because that implies it's an actualy test. (Like a unittest.) Maybe `md_sample`?

Also, it appears to be dead code?